get_that_data.py

```diff
@@ -1,41 +1,7 @@
 from src.data.data_fetcher import DataFetcher
 
-
-
-fetcher = DataFetcher("2020", "P")
-print(fetcher.api_starting_url_container)
-print(fetcher.gimmie_data(record_limit=5))
-
-
-#### Need to write this into the other file
-
-# Create instance of data_fetcher class class
-# fec = data_fetcher(starting_url, complete_list)
-
-# # Inizialize get_the_data to work
-# work = get_the_data(fec)
-
-# # Starts looping over all transactions
-# work.gimmie_data()
-
-# # Once complete_list has all transaction we create a pandas dataframe
-# df = pd.DataFrame(
-#     complete_list,
-#     columns=[
-#         "contributor_occupation",
-#         "contributor_employer",
-#         "contributor_city",
-#         "contributor_state",
-#         "contributor_zip",
-#         "party",
-#     ],
-# )
-
-# Saves DataFrame as serialized object, will import this to ML program
-# Add path and uncommet to save pickled data
-# df.to_pickle("./pickled_data.pkl")
-
-# Prints DataFrame, be careful, this can get bigggggg
-# print(df)
-
-# Thanks for reading my Ted Talk
\ No newline at end of file
+fetcher = DataFetcher("2020", "S")
+fetcher.api_starting_url_container
+fetcher.gimmie_data(record_limit=6)
+fetcher._build_df()
```

Comment:
The whole point of the leading `_` is so that end-users aren't even aware that these function exist.

Basically there are two key ideas:
1. You want each of your functions to do one thing and do it well. This potentially means a ton of little functions, but that's okay because
    * The end user doesn't have to see the "clutter"
    * Because the functions are very well-defined in what they do, it's easier to write tests for them

2. All of the `_hidden_functions()` should be called behind the scenes, so everything just works, automagically.

I'll be more explicit in the other file, but my gut says `._build_df()` should be called at the very end of `gimmie_data()`